terracumber-cli

```diff
@@ -95,6 +94,10 @@ def parse_args():
     parser.add_argument('--bastion_ssh_key', help="""Bastion key to be use""", dest='bastion_ssh_key', default=None)
     parser.add_argument('--bastion_user', help="""Bastion user to be use""", dest='bastion_user', default='ec2-user')
     parser.add_argument('--bastion_hostname', help="""Bastion hostname to use""", dest='bastion_hostname', default=None)
+    parser.add_argument('--use-minion-cleaner', help='Activate or deactivate the minion cleaner',
```

Comment:
In Terracumber code, we might want to abstract the concept of Salt Minions, that would make it more flexible if we decide to handle other kind of components.
What about naming it `--use-resource-cleaner`?